lib/gui/modules/drive-scanner.js

```diff
@@ -40,19 +40,6 @@ const availableDrives = Rx.Observable.timer(
     return Rx.Observable.fromNodeCallback(drivelist.list)();
   })
 
-  // Build human friendly "description"
-  .map((drives) => {
-    return _.map(drives, (drive) => {
-      drive.name = drive.device;
-
-      if (os.platform() === 'win32' && !_.isEmpty(drive.mountpoints)) {
```

Comment:
If I see this correctly, this block can now also be removed from https://github.com/resin-io/etcher/pull/1637, right @jviotti?